File: app/model/OverviewEntryModel.scala
package model

case class OverviewEntryModel(teamHeader: String, name: String, picture: String, colour: String){
/*    override def equals(that: Any) : Boolean = {
        that match {
          case over : OverviewEntryModel =>
              this.name == over.name &&
              this.teamHeader == over.teamHeader &&
              this.picture == over.teamHeader &&
              this.colour == over.colour
          case _ => false
        }
  }
  override def hashCode : Int = {
      com.google.common.base.Objects.hashCode(
          this.teamHeader, this.name, this.picture, this.colour
      )
  }*/
  }

File: app/model/OverviewEntryModel.java
package model;

public class OverviewEntryModel {
    public final String teamHeader;
    public final String name;
    public final String picture;
    public final String colour;
    public final String link;

    public OverviewEntryModel(String teamHeader, String name, String picture, String colour, String link) {
        this.teamHeader = teamHeader;
        this.name = name;
        this.picture = picture;
        this.colour = colour;
        this.link = link;
    }
}